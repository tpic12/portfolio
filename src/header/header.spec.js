import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Header from "./header";

describe("<Landing />", () => {
  it("renders by default", () => {
    const wrapper = shallow(<Header />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
