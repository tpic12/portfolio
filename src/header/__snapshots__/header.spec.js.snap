// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Landing /> renders by default 1`] = `
<nav
  className="navbar navbar-hidden"
  role="navigation"
>
  <div
    id="menuToggle"
  >
    <input
      checked={false}
      onChange={[Function]}
      onClick={[Function]}
      type="checkbox"
    />
    <span />
    <span />
    <span />
    <ul
      id="menu"
    >
      <a
        href="#about-section"
        onClick={[Function]}
      >
        <li>
          About
        </li>
      </a>
      <a
        href="#projects-section"
        onClick={[Function]}
      >
        <li>
          Projects
        </li>
      </a>
      <a
        href="#contact-section"
        onClick={[Function]}
      >
        <li>
          Contact
        </li>
      </a>
    </ul>
  </div>
  <div
    className="navbar--logo-holder white"
  >
    <h1>
      {tP}
    </h1>
  </div>
</nav>
`;
